migrations/versions/840_add_application_close_date_for_framework.py

```diff
@@ -0,0 +1,31 @@
+"""Add `application_close_date` for Framework
+
+Revision ID: 840
+Revises: 830
+Create Date: 2017-02-06 11:09:26.852142
+
+"""
+from alembic import op
+from datetime import date
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '840'
+down_revision = '830'
+
+
+
+def upgrade():
+    op.add_column('frameworks', sa.Column('application_close_date', sa.Date(), nullable=True))
```

Comment:
All our other timestamps are `sa.DateTime()` - I know we only need the Date for the current use-case, but feels like it wouldn't hurt to keep it the same here.

In terms of the time of day to set - I think DOS closed at 3pm, G8 and DOS2 at 5pm.